src/main.d

```
@@ -260,7 +260,9 @@ string[] expandArgs(string[] args)
 	if (args.length == 1)
 		args ~= ".";
 	foreach (arg; args[1 ..$])
-	{
+	{        
+		if (!exists(arg))
```

I'm not sure that failing silently is the best idea.